source/time_tables/tests/route_schedules_test.cpp

```diff
@@ -893,3 +893,152 @@ BOOST_AUTO_TEST_CASE(route_schedule_multiple_days) {
         BOOST_CHECK_EQUAL(route_schedule.table().rows(1).date_times(4).time(), "08:05"_t);
     }
 }
+
+// stop_times with skipped_stops from the begining (*)
+//      VJ_A   VJ_B   VJ_C  VJ_D
+// st1  1       4       3   5 *
+// st2  2       4 *     4   5 *
+// st3  3       4 *     5   6
+// st4  4       5       6   7
+// st5  5       6       7   8
+
+// wanted but not obtained:
+//      VJ_A   VJ_B   VJ_C  VJ_D
+// st1  1       4       3   -
+// st2  2       -       4   -
+// st3  3       -       5   6
+// st4  4       5       6   7
+// st5  5       6       7   8
+
+// obtained: here vehicle_journey VJ_D disappeared as the start date_time is skipped_stop
+//      VJ_A   VJ_C   VJ_B
+// st1  1       3       4
+// st2  2       4       -
+// st3  3       5       -
+// st4  4       6       5
+// st5  5       7       6
+
+BOOST_AUTO_TEST_CASE(stop_times_with_skipped_stops_from_the_begining) {
```

Comment:
If it stays like this for a bit:
```suggestion
BOOST_AUTO_TEST_CASE(xfail_stop_times_with_skipped_stops_from_the_begining) {
```

Reply:
:+1: